pages/index.js

```diff
@@ -67,26 +104,77 @@ export default function Home({ posts, status }) {
             </div>
           </div>
         </div>
-        <hr className="mb-10" />
-        <div className="pl-3 my-10 !w-80  rounded-[10px] !bg-gray-100">
-          <span>
-            <i className="w-16 h-16 rounded-full text-gray-500 cursor-pointer">
-              <FontAwesomeIcon
-                icon={faMagnifyingGlass}
-                className="pr-1 text-sm"
-              />
-            </i>
-          </span>
-          <input
-            className="!border-0 !bg-gray-100"
-            placeholder="Search Blogs"
-            type="text"
-            value={keyword}
-            onChange={(e) => {
-              setKeyword(e.target.value);
-              setResults(searchBlogs(posts, e.target.value));
-            }}
-          />
+
+        <div className="flex flex-col lg:flex-row space-y-8 lg:space-y-0 lg:justify-between lg:items-center mb-16">
+          <div className="lg:basis-8/12 h-10 border-b border-[#e6e6e6]">
+            <Swiper
+              navigation={true}
+              onClick={(swiper) => setActiveIndex(swiper.clickedIndex)}
+              modules={[Navigation]}
+              breakpoints={{
+                0: {
+                  slidesPerView: 2,
+                },
+                576: {
+                  slidesPerView: 3,
+                },
+                1200: {
+                  slidesPerView: 5,
+                },
+              }}
+            >
+              <SwiperSlide
+                data-history="slide 0"
```

Comment:
What is use of it?

Reply:
i don't know , i thought it part of swiper syntax.